packages/wdio-browserstack-service/src/util.ts

```diff
@@ -0,0 +1,64 @@
+import { BROWSER_DESCRIPTION } from './constants'
+
+type GenericBrowserCaps = {
+    [key: string]: { capabilities: BrowserCapabilities };
+};
+
+type BrowserCapabilities = GenericBrowserCaps & {
```

Comment:
We should use `WebDriver.Capabilities` types here instead of defining something custom

Reply:
Thanks for the tip! I gave it another round using `WebDriver.Capabilities` and `WebdriverIO.BrowserObject`, however two things I'm still not sure:

1. Not all keys from `BROWSER_DESCRIPTION` are within `Webdriver.Capabilities` so I had to included that in the local type.

2. `getBrowserCapabilities` receives a `string` `browserName` used to index `browser` and `caps` to get a `Webdriver.Capabilities` object. The way I found to solve it was through the union with 
```
{
    [key: string]: { capabilities: Webdriver.Capabilities };
}
```

What do you think?